test/e2e/e2e-test/docker/storage/docker-compose.influxdb.yml

```diff
@@ -22,6 +22,8 @@ services:
       - 8086
     networks:
       - e2e
+    volumes:
+      - /var/run/influxdb2:/var/lib/influxdb
```

Comment:
This should not be committed, the directory name (`/var/run/influxdb2`)l depends on the host machine.